CHANGELOG.md

```diff
@@ -4,15 +4,20 @@ All notable changes to this project will be documented in this file.
 Please follow the [guidance](#adding-to-this-changelog) at the bottom of this file when making changes
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
-and follows a [Backwards Compatability Policy](https://docs.solana.com/developing/backwards-compatibility)
+and follows a [Backwards Compatibility Policy](https://docs.solana.com/developing/backwards-compatibility)
 
-Release channels have their own copy of this changelog:
-* [edge - v1.17](#edge-channel)
-* [beta - v1.16](https://github.com/solana-labs/solana/blob/v1.16/CHANGELOG.md)
-* [stable - v1.14](https://github.com/solana-labs/solana/blob/v1.14/CHANGELOG.md)
+## [1.17.3] - Unreleased
+* Changes
+* Upgrade Notes
+
+## [1.17.2]
+* Upgrade Notes
+  * `--no-skip-initial-accounts-db-clean` flag may help <=128GB nodes avoid OOM on startup.
```

Comment:
this flag was introduced hidden so as not to promote its use. we do not want to support it in the long term. calling it out here is probably counterproductive to that goal

Reply:
Fair point. I've removed it.